dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/configMLDT/configForEutaxonomyFinancialsMLDT.ts

```diff
@@ -71,42 +97,45 @@ export const configForEuTaxonomyFinancialsMLDT = [
         type: "cell",
         label: euTaxonomyKpiNameMappings.fiscalYearDeviation,
         explanation: euTaxonomyKpiInfoMappings.fiscalYearDeviation,
-        shouldDisplay: (): boolean => true,
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean => !!dataset.fiscalYearDeviation,
         valueGetter: plainStringValueGetterFactory("fiscalYearDeviation"),
       },
       {
         type: "cell",
         label: euTaxonomyKpiNameMappings.fiscalYearEnd,
         explanation: euTaxonomyKpiInfoMappings.fiscalYearEnd,
-        shouldDisplay: (): boolean => true,
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean => !!dataset.fiscalYearEnd,
         valueGetter: plainStringValueGetterFactory("fiscalYearEnd"),
       },
       {
         type: "cell",
         label: euTaxonomyKpiNameMappings.scopeOfEntities,
         explanation: euTaxonomyKpiInfoMappings.scopeOfEntities,
-        shouldDisplay: (): boolean => true,
-        valueGetter: yesNoValueGetterFactory(euTaxonomyKpiNameMappings.scopeOfEntities),
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean =>
+          shouldValueBeDisplayedForYesNo(dataset.scopeOfEntities),
+        valueGetter: yesNoValueGetterFactory("scopeOfEntities"),
       },
       {
         type: "cell",
         label: euTaxonomyKpiNameMappings.nfrdMandatory,
         explanation: euTaxonomyKpiInfoMappings.nfrdMandatory,
-        shouldDisplay: (): boolean => true,
-        valueGetter: yesNoValueGetterFactory(euTaxonomyKpiNameMappings.nfrdMandatory),
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean =>
+          shouldValueBeDisplayedForYesNo(dataset.nfrdMandatory),
+        valueGetter: yesNoValueGetterFactory("nfrdMandatory"),
       },
       {
         type: "cell",
         label: euTaxonomyKpiNameMappings.euTaxonomyActivityLevelReporting,
         explanation: euTaxonomyKpiInfoMappings.euTaxonomyActivityLevelReporting,
-        shouldDisplay: (): boolean => true,
-        valueGetter: yesNoValueGetterFactory(euTaxonomyKpiNameMappings.euTaxonomyActivityLevelReporting),
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean =>
+          shouldValueBeDisplayedForYesNo(dataset.euTaxonomyActivityLevelReporting),
+        valueGetter: yesNoValueGetterFactory("euTaxonomyActivityLevelReporting"),
       },
       {
         type: "cell",
         label: euTaxonomyKpiNameMappings.numberOfEmployees,
         explanation: euTaxonomyKpiInfoMappings.numberOfEmployees,
-        shouldDisplay: (): boolean => true,
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean => !!dataset.numberOfEmployees,
```

Comment:
This test would also hide the info if numberOfEmployees = 0

Reply:
Changed